### DISCUSSION TITLE
Avoid these objects to get detected

### DISCUSSION THREAD

**Discussion by Produce2727** [blakeblackshear/frigate#5017] [Ask A Question]
Hello what settings I should tweak so that these objects for example the rain get not detected?

![2023-01-11 11_39_54-Frigate – Home Assistant](https://user-images.githubusercontent.com/42067245/211802682-4b38ccac-5d3e-41fe-b8dd-15beef1a1349.png)
![2023-01-11 11_40_40-Frigate – Home Assistant](https://user-images.githubusercontent.com/42067245/211802686-66a38b74-571f-447e-9610-396b8404f9c6.png)
![2023-01-11 11_41_11-Frigate – Home Assistant](https://user-images.githubusercontent.com/42067245/211802689-591f5a6f-c32d-4a76-a5d4-aeeae49c8d8a.png)

  garten_2:
    ffmpeg:
      inputs:
        - path: ...
          roles:
            - detect
            - rtmp
        - path: ...
          roles:
            - record
    detect:
      width: 2560
      height: 1440
      fps: 5

    record:
      enabled: True
      events:
        retain:
          default: 90

    snapshots:
      enabled: True
      retain:
        default: 90

    objects:
      track:
        - person
        - cat
        - dog
      filters:
          person:
            mask:
              - 296,236,296,268,255,269,251,236

    motion:
      mask:
        - 167,339,169,360,94,360,93,339

**Comment by NickM-27** [comment]
The first one probably a min_area filter. Second one use an object filter. The last one use a max_area filter.
